utils/parametric/_library_client.py

```diff
@@ -829,6 +829,23 @@ def get_tracer_config(self) -> Dict[str, Optional[str]]:
             "dd_trace_rate_limit": config_dict.get("dd_trace_rate_limit", None),
         }
 
+    def container_exec_run(self, command: str) -> tuple[bool, str]:
```

Comment:
Hi @mtoffl01 ,

What is the purpose of this method ?

Reply:
It's required to fulfill the Test_Config_TraceLogDirectory test. The original PR to implement this test ([here](https://github.com/DataDog/system-tests/pull/3012/files)) only configured the function on the HTTP Client; so I've added it to the GRPC client (which golang uses).